service/tests/integration/loader.rs

```diff
@@ -136,13 +136,18 @@ async fn sqlx_pool() {
 
     let log = rx.recv().await.unwrap();
     assert_eq!(log.deployment_id, deployment_id);
+
+    let fields: serde_json::Map<String, serde_json::Value> =   
```

Comment:
At a high level, if this test passes, then the implementation should be all good.

If you want inspiration for a lower level, isolated test then you can try to create a test which registers just the tracing layer, logs a bunch of events you want to test (in the test function), and then confirm/assert those events were captured by the layer correctly :man_shrugging:

Reply:
Thanks! I did make a test just like that, checking them at the receiving end of the channel.